src/ui/component/fileViewer/view.jsx

```diff
@@ -76,7 +76,7 @@ class FileViewer extends React.PureComponent<Props> {
     }
 
     this.handleAutoplay(this.props);
-    window.addEventListener('keydown', this.handleKeyDown);
+    // window.addEventListener('keydown', this.handleKeyDown);
```

Comment:
is this functionality being removed? maybe comment out the function too and explain why if it's anticipated that it returns?

Reply:
@jessopb I'm fixing most of my own comments but I don't know what or why this was changed, so I'm leaving as is